db/migrate/20230422114809_create_feedbacks.rb

```diff
@@ -1,9 +1,9 @@
 class CreateFeedbacks < ActiveRecord::Migration[6.1]
```

Comment:
Since I didn’t merge it, I didn’t complicate this and roll additional null constraints migration, so I decided to delete the table and make a new one.